easybuild/easyblocks/generic/makecp.py

```diff
@@ -47,14 +47,18 @@ def extra_options(extra_vars=None):
         Define list of files or directories to be copied after make
         """
         extra = {
-            'files_to_copy': [[], "List of files or dirs to copy", MANDATORY],
+            'files_to_copy': [None, "List of files or dirs to copy", MANDATORY],
             'with_configure': [False, "Run configure script before building", BUILD],
         }
         if extra_vars is None:
             extra_vars = {}
         extra.update(extra_vars)
         return ConfigureMake.extra_options(extra_vars=extra)
 
+    def __init__(self, *args, **kwargs):
```

Comment:
is this needed?

Reply:
It's not, I was planning something else first, and then overlook not actually doing anything in the constructor

Fixed in 0f6992e8e